packages/i/imgui/port/xmake.lua

```diff
@@ -16,6 +17,10 @@ if has_config("glfw_opengl3") then
     if has_config("use_glad") then
         add_requires("glad")
     end
+elseif has_config("glfw_vulkan") then
+    add_requires("glfw")
+    add_requires("vulkansdk")
```

Comment:
xmake-repo should not have the package vulkansdk, only vulkan-loader

did you test it on your host?

Reply:
I have tested on Window. `add_requires` has builin find_package script support right?

Reply:
right, I forgot it.